src/Input.tsx

```
@@ -94,6 +94,7 @@ const Input = forwardRef<InputRef, InputProps>((props, ref) => {
   }));
 
   useEffect(() => {
+    keyLockRef.current && (keyLockRef.current = false);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)